android/build.gradle

```
@@ -48,6 +48,7 @@ repositories {
 
 dependencies {
     implementation "com.facebook.react:react-native:${safeExtGet('reactNativeVersion', '+')}"
+    implementation "com.android.installreferrer:installreferrer:1.1"
```

could alter this to be a variable just in case people need to override?

```suggestion
    implementation "com.android.installreferrer:installreferrer:${safeExtGet('installReferrerVersion', '1.1')}"
```

I think